pom.xml

```diff
@@ -82,8 +82,8 @@
     <module>mod-copycat</module>
     <module>mod-bulk-operations</module>
     <module>mod-consortia</module>
-<!--    <module>mod-dcb</module>-->
-<!--    <module>edge-dcb</module>-->
+    <module>mod-dcb</module>
```

Comment:
Not sure if this is okay.  
@gurleenkaurbp :- is mod-dcb avilable in karate env. if not this will cause an issue i think.

Reply:
Disabled both modules.